docker-compose.yml

```diff
@@ -1,24 +1,49 @@
 version: '3'
 services:
-  web:
-    build: .
+
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile
```

Comment:
Are you using long `build` syntax to highlight that default `Dockerfile` is being used?

Reply:
When we have 2 Dockerfile's, I want to inject it very clearly